etlas-cabal/etlas-cabal.cabal

```diff
@@ -1,6 +1,6 @@
 name:          etlas-cabal
 -- @VERSION
-version:       1.4.0.1
```

Comment:
Not sure if a new version is needed cause it usually involves update the binaries in downloading servers, right @rahulmutt?

Reply:
The binaries are manually uploaded - we haven't setup an automated CI system yet, so this is OK.